ext/json/json.c

```diff
@@ -95,6 +96,9 @@ static PHP_MINIT_FUNCTION(json)
 	INIT_CLASS_ENTRY(ce, "JsonSerializable", json_serializable_interface);
 	php_json_serializable_ce = zend_register_internal_interface(&ce);
 
+	INIT_CLASS_ENTRY(ce, "JsonException", NULL);
```

Comment:
IMHO `JsonParseException` suites better for `json_decode`, because it's parsing related exception, and for `json_encode` it would be for eg. `JsonEncodeException` or `JsonSerializeException`